molo/polls/migrations/0003_create_polls_index_pages.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def create_polls_index(apps, schema_editor):
+    from molo.core.models import Main
+    from molo.polls.models import Question, FreeTextQuestion, PollsIndexPage
+    main = Main.objects.all().first()
+
+    if main:
+        polls_index = PollsIndexPage(title='Polls', slug='polls')
+        main.add_child(instance=polls_index)
+        polls_index.save_revision().publish()
+
+        # Move existing Questions
```

Comment:
I think this migration should stop here, and the rest should be handled on an app level